DOC_CHANGES.md

```diff
@@ -7,3 +7,69 @@ Description of the required change.
 -->
 
 # Ohai Doc Changes:
+
+## Shard Seed Plugin
+
+The new `shard_seed` attribute provides a normally distributed integer ID value
+to all nodes. This value will be between 0 and 4294967295 (0xFFFFFFFF), based
+on a consistent hash of one or more pieces of node data. A consistent hash
+means that a node with the same source inputs will always result in the same
+shard seed, but collisions are possible.
+
+The sources used to compute the seed value can be configured either in your
+Chef config file or your Ohai config file ([ed: does anyone use Ohai config files?]):
```

Comment:
Think it's worth mentioning ohai config files at all for normal users though? Probably a red herring for most people that are smart and don't touch ohai core :)